src/coreclr/vm/corelib.h

```diff
@@ -330,11 +330,12 @@ DEFINE_FIELD_U(_xptrs,             ExceptionObject,    _xptrs)
 DEFINE_FIELD_U(_xcode,             ExceptionObject,    _xcode)
 DEFINE_FIELD_U(_HResult,           ExceptionObject,    _HResult)
 DEFINE_CLASS(EXCEPTION,             System,                 Exception)
+DEFINE_METHOD(EXCEPTION,            INTERNAL_PRESERVE_STACK_TRACE, InternalPreserveStackTrace, IM_RetVoid)
+// Following Exception members are only used when FEATURE_COMINTEROP
```

Comment:
Instead of this comment can we use the `#ifdef FEATURE_COMINTEROP` mechanism?

Reply:
The comment in this file says that ILLink can only process the markers at type level:
https://github.com/dotnet/runtime/blob/20e5237104ddd178f12bc501352e45a5e91e5048/src/coreclr/vm/corelib.h#L71-L79